### PR TITLE
Update libs

### PR DESCRIPTION
**Pull request by altro3**

### CODE REVIEW

**Comment by altro3** [comment]
Ignore  some tests because of this: https://github.com/micronaut-projects/micronaut-core/issues/10733

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautJavaCodegen.java [hunk 1]
```diff
@@ -1071,7 +1071,7 @@ private String makeSureImported(String typeName, Set<String> imports) {
     public String toVarName(String name) {
         var varName = super.toVarName(name);
 
-        if (varName.chars().allMatch(Character::isUpperCase)) {
+        if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_')) {
```
Comment (sdelamo) [inline openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautJavaCodegen.java:1074]: why are you adding these changes to a PR about updating versions? Mixing changes makes reviews really hard.

File: test-suite-java-server-generator/src/test/groovy/io/micronaut/openapi/test/api/ResponseBodyControllerSpec.groovy [hunk 2]
```diff
@@ -157,6 +157,7 @@ class ResponseBodyControllerSpec extends Specification {
         HttpStatus.NOT_FOUND == e.status
     }
 
+    @Ignore("https://github.com/micronaut-projects/micronaut-core/issues/10733")
```
Comment (sdelamo) [inline test-suite-java-server-generator/src/test/groovy/io/micronaut/openapi/test/api/ResponseBodyControllerSpec.groovy]: Use `@PendingFeature` instead `@Ignore`.

File: openapi-generator/src/test/resources/3_0/controller-enum.yml [hunk 3]
```diff
@@ -103,6 +103,7 @@ paths:
           enum:
             - "year"
             - "rating"
+            - "big_rating"
```
Comment (sdelamo) [inline openapi-generator/src/test/resources/3_0/controller-enum.yml:106]: why is this change necessary in the PR to update versions?

File: openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java [hunk 4]
```diff
@@ -105,7 +105,7 @@ public static MyEnum valueOf(
 
   private final int value;
 
-  private MyEnum(int value) {
```
Comment (sdelamo) [inline openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java]: why is this change necessary in the PR to update versions?

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java [hunk 5]
```diff
@@ -867,7 +866,7 @@ public String getTypeDeclaration(Schema p) {
         Schema<?> schema = unaliasSchema(p);
         Schema<?> target = ModelUtils.isGenerateAliasAsModel() ? p : schema;
         if (ModelUtils.isArraySchema(target)) {
-            Schema<?> items = getSchemaItems((ArraySchema) schema);
+            Schema<?> items = ModelUtils.getSchemaItems(schema);
```
Comment (sdelamo) [inline openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java:869]: why is this change necessary in the PR to update versions?

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java [hunk 6]
```diff
@@ -1187,7 +1186,7 @@ private String makeSureImported(String typeName, Set<String> imports) {
     public String toVarName(String name) {
         var varName = super.toVarName(name);
 
-        if (varName.chars().allMatch(Character::isUpperCase)) {
+        if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_')) {
```
Comment (sdelamo) [inline openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java:1189]: why is this change necessary in the PR to update versions?

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautJavaCodegen.java (hunk 1 above)
Comment (altro3) [reply openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautJavaCodegen.java:1074]: because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0

File: openapi-generator/src/test/resources/3_0/controller-enum.yml (hunk 3 above)
Comment (altro3) [reply openapi-generator/src/test/resources/3_0/controller-enum.yml:106]: this case need to test openapi-generator 7.5.0. Problem ws with generating enums, which contains `_` symbol.

`if (varName.chars().allMatch(c -> Character.isUpperCase(c) || c == '_'))` - fix it

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java (hunk 5 above)
Comment (altro3) [reply openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java:869]: because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0

File: openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java (hunk 6 above)
Comment (altro3) [reply openapi-generator/src/main/java/io/micronaut/openapi/generator/AbstractMicronautKotlinCodegen.java:1189]: because this fix for openapi-generator 7.5.0, without this fix you can't use 7.5.0

File: openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java (hunk 4 above)
Comment (altro3) [reply openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java]: not necessary

File: test-suite-java-server-generator/src/test/groovy/io/micronaut/openapi/test/api/ResponseBodyControllerSpec.groovy (hunk 2 above)
Comment (altro3) [reply test-suite-java-server-generator/src/test/groovy/io/micronaut/openapi/test/api/ResponseBodyControllerSpec.groovy]: fixed

File: openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java (hunk 4 above)
Comment (altro3) [reply openapi/src/test/groovy/io/micronaut/openapi/proto/MyEnum.java]: reverted

**Review by sdelamo — APPROVED** [review]
